aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/actions/SignInChallengeCognitoActions.kt

```diff
@@ -26,36 +31,41 @@ import com.amplifyframework.statemachine.codegen.events.SignInChallengeEvent
 import com.amplifyframework.statemachine.codegen.events.SignInEvent
 
 object SignInChallengeCognitoActions : SignInChallengeActions {
-    override fun verifySignInChallenge(
+    override fun verifyChallengeAuthAction(
         event: SignInChallengeEvent.EventType.VerifyChallengeAnswer,
         challenge: AuthChallenge
-    ) = Action<AuthEnvironment>("VerifySignInChallenge") { id, dispatcher ->
+    ): Action = Action<AuthEnvironment>("VerifySignInChallenge") { id, dispatcher ->
         logger?.verbose("$id Starting execution")
         val evt = try {
             val username = challenge.username
-            var challengeResponses = mapOf("USERNAME" to username)
+            var challengeResponses = mapOf<String, String>()
```

Comment:
rePost Tyler's comment on this PR:
Let's make `challengeResponses` a val and mutableMap.